charts/kafka-operator/templates/operator-deployment.yaml

```diff
@@ -34,8 +34,8 @@ spec:
         app: prometheus
         component: alertmanager
     spec:
-      {{- if .Values.rbac.enabled }}
-      serviceAccountName: {{ include "kafka-operator.fullname" . }}-operator
+      {{- if .Values.serviceAccount.name }}
```

Comment:
you also added `.Values.serviceAccount.enabled` shouldn't that flag be used here?

Reply:
OK I think I was confused - the `.Values.serviceAccount.enabled` is actually ``.Values.serviceAccount.create`